.cirrus/Dockerfile.ubuntu14.04

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:trusty-20181115
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN set -e -x ;\
+    apt-get -y update ;\
+    apt-get -y upgrade ;\
+    apt-get -y install build-essential clang gperf bison flex libreadline-dev gawk \
+                       tcl-dev libffi-dev git graphviz xdot pkg-config python autoconf
+
+RUN set -e -x ;\
+    cd /tmp ;\
+    git clone git://github.com/steveicarus/iverilog.git ;\
+    cd iverilog ;\
+    autoconf ;\
+    ./configure ;\
+    make -j $(nproc) ;\
+    make install ;\
+    cd .. ;\
+    rm -rf iverilog
```

Comment:
Is... is this really how you are supposed to use Docker? Is this *really* better than Travis?..

Reply:
Yes.